configure.ac

```diff
@@ -345,6 +346,7 @@ if test "x$CXXFLAGS_overridden" = "xno"; then
   AX_CHECK_COMPILE_FLAG([-Wredundant-decls],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wredundant-decls"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wunused-variable],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wunused-variable"],,[[$CXXFLAG_WERROR]])
   AX_CHECK_COMPILE_FLAG([-Wdate-time],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wdate-time"],,[[$CXXFLAG_WERROR]])
+  AX_CHECK_COMPILE_FLAG([-Wsign-compare],[WARN_CXXFLAGS="$WARN_CXXFLAGS -Wsign-compare"],,[[$CXXFLAG_WERROR]])
```

Comment:
It is not necessary to add `-Wsign-compare` because it is enabled by `-Wall` in gcc and by `-Wextra` in clang. Just a few lines above we enable both `-Wall` and `-Wextra`.

https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html#index-Wsign-compare
https://clang.llvm.org/docs/DiagnosticsReference.html#wextra

This was explicitly requested by @practicalswift, but maybe he did not realize that it is already enabled in both gcc and clang?

Reply:
Leaving as it seems harmless to me.